R/bfast01.R

```diff
@@ -1,3 +1,21 @@
+# bfast01 - Checking for one major break in the time series
+# Copyright (C) 2019  Jan Verbesselt, Dainius Masiliūnas, Dongdong Kong
```

Comment:
@verbe039 Can you check who should be credited for most of these files, like bfast01 etc.?